src/main/java/kitchenpos/application/TableGroupService.java

```diff
@@ -29,20 +30,34 @@ public TableGroupService(final OrderDao orderDao, final OrderTableDao orderTable
     }
 
     @Transactional
-    public TableGroup create(final TableGroup tableGroup) {
-        final List<OrderTable> orderTables = tableGroup.getOrderTables();
+    public TableGroupDto create(final TableGroupCreateRequest tableGroup) {
 
-        if (CollectionUtils.isEmpty(orderTables) || orderTables.size() < 2) {
-            throw new IllegalArgumentException();
+        final List<Long> orderTableIds = tableGroup.getOrderTableIds();
+        final List<OrderTable> savedOrderTables = orderTableDao.findAllByIdIn(orderTableIds);
```

Comment:
orderTableIds 변수는 변수로 추출하지 않아도 의미 전달이 가능하기 때문에 하나로 합치면 더 가독성을 향상시킬 수 있을 듯 합니다~!

```suggestion
        final List<OrderTable> savedOrderTables = orderTableDao.findAllByIdIn(tableGroup.getOrderTableIds());
```

Reply:
너무 길어질까봐 나눠놨는데 다른곳에서 사용하는게 없으니 하나로 합치도록 하겠습니다.